build: change the sample values for ui templates (#1259)
SwissDataScienceCenter/renku: charts/values.yaml.changelog.md

* `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`

----
## Changes on top of Renku 0.6.5
* DELETE - *ui.templatesRepository*

> * NEW - *ui.templates*: the UI project creation has been overhauled and the settings are stored here.

Reviewer: ```suggestion
* NEW - *ui.templates*: the UI project creation has been overhauled and can be configured here.
```

* `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`

----

> ## Changes on top of Renku 0.7.0

Reviewer: ```suggestion
## Upgrading to Renku 0.8.0 (INCLUDES BREAKING CHANGES!)
```
I guess this should go into a `0.8.0` release as it requires changes to the values file, right?

Author: I thought the version 0.7.0 is yet to be released, right? Should I move these changes in the already existing 0.7.0 section?

Reviewer: ah yeah, ofc, sorry. Yes, just add it to the 0.7.0 section 👍
